lib/kontena/plugin/vagrant/master/create_command.rb

```diff
@@ -23,6 +25,18 @@ def execute
       )
     end
 
+    def ask_instance_memory
```

Comment:
Actually it's fine in this plugin, more control is needed with other plugins.. will cleanup this a little bit.